minigalaxy/ui/categoryfilters.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+from minigalaxy.logger import logger
+from minigalaxy.translation import _
+from minigalaxy.paths import UI_DIR
+from minigalaxy.ui.filterswitch import FilterSwitch
+from minigalaxy.ui.gtk import Gtk
+
+
+@Gtk.Template.from_file(os.path.join(UI_DIR, "categoryfilters.ui"))
+class CategoryFilters(Gtk.Dialog):
+    __gtype_name__ = "CategoryFilters"
+
+    genre_filtering_grid = Gtk.Template.Child()
+    filter_dict = {}
+    categories = [
```

Comment:
Are these the standard categories for GOG? Do you have a source for these?

Reply:
The source for these categories is the list in the lower left of this GOG store page: https://www.gog.com/en/games

![image](https://github.com/sharkwouter/minigalaxy/assets/1861493/6abe5038-da68-499b-9f75-6d4b2615ef8c)